components.html

```diff
@@ -0,0 +1,449 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Component-Library</title>
+    <link href="style.css" rel="stylesheet" />
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" rel="stylesheet"/>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/js/bootstrap.min.js"></script>
+</head>
+<body>
+    <nav class="navigation container">
+        <div class="nav-brand"></div>
+        <ul class="list-non-bullet nav-pills">
+            <li class="list-item-inline-components">
+                <a class="link" href="/components.html">Components</a>
+            </li>
+            <li class="list-item-inline-github">
+                <a class="link" href="https://github.com/mansiagarwal451/My-Component-library">Github</a>
+            </li>
+           
+        </ul>
+    </nav>
+    <hr>
+    <!-- Avatar -->
+    <h1>Avatar</h1>
+    <br/>
+    <div class="avatar-container"><img src="https://cdn.pixabay.com/photo/2014/02/27/16/10/tree-276014__340.jpg" alt="Avatar" class="avatar avatar--s"></div>
+    <div class="avatar-container"><img src="https://cdn.pixabay.com/photo/2014/02/27/16/10/tree-276014__340.jpg" alt="Avatar" class="avatar avatar--m"></div>
+    <div class="avatar-container"><img src="https://cdn.pixabay.com/photo/2014/02/27/16/10/tree-276014__340.jpg" alt="Avatar" class="avatar avatar--l"></div>
+    <br />
```

Comment:
It's good practice to avoid using ```br``` and ```hr``` tags here as they don't have any semantic value to the document.
For ref: [stackoverflow](https://stackoverflow.com/questions/462619/should-br-and-hr-be-avoided-at-all-costs-in-web-design/462703#:~:text=br%20elements%20must%20not%20be,thematic%20groups%20in%20a%20paragraph.&text=can%20very%20well,when%20to%20use%20these%20elements.)